cfn-resources/util/progressevent/failed_event.go

```diff
@@ -23,6 +23,8 @@ import (
 
 func getHandlerErrorCode(response *http.Response) string {
 	switch response.StatusCode {
+	case http.StatusConflict:
+		return cloudformation.HandlerErrorCodeAlreadyExists
```

Comment:
this is a complex case ghovardan, we avalueted mapping the conflict with the AlreadyExists before, we reialised that the StatusConflict is not always an Already exists, and each resource handles the AlreadyExists in different ways, some of them returns a 400 others a 200 with an empty body, so this might work for this resource, but not for others, this case we should handle en each resource

Reply:
Ok, I ll move this to Resource level.